report_thesis/src/sections/introduction.tex

```diff
@@ -1,19 +1,20 @@
 \section{Introduction}\label{sec:introduction}
-The NASA Viking missions in the 1970s were the first to successfully land a rover on Mars, aiming to determine if life existed on the planet.
-One experiment suggested the presence of life, but the results were ambiguous and inconclusive, and NASA was unable to repeat the experiment due to budget constraints\cite{marsnasagov_vikings}.
+The NASA Viking missions in the 1970s were the first to successfully land on Mars, aiming to determine if life existed on the planet.
+One experiment suggested the presence of life, but the results were ambiguous and inconclusive, and NASA was unable to repeat the experiment.
+Nevertheless, these missions were deemed a monumental success and advanced our knowledge of the Martian environment.\cite{marsnasagov_vikings}
 
-A few decades later, the philosophy of Martian exploration had shifted from searching for life to investigating whether Mars ever had the conditions to support life as we know it.
-The \gls{mer} mission, which included the Spirit and Opportunity rovers, discovered clear evidence that water once flowed on Mars.
-As water alone is not enough to support life, NASA shifted their focus to search for organic material as well\cite{marsnasagov_observer, marsnasagov_spirit_opportunity}.
+Leveraging the knowledge gained from the Viking missions, NASA launched the \gls{mer} mission in 2003 to further whether Mars ever had the conditions to support life as we know it.
+The mission landed two rovers, Spirit and Opportunity, on Mars in January 2004, and they quickly discovered clear evidence that water once flowed on Mars.
+However, since water alone is not enough to support life, the next objective was to search for organic material as well.\cite{marsnasagov_observer, marsnasagov_spirit_opportunity}
 
 The Curiosity rover landed on Mars in August 2012 inside Gale Crater as part of the \gls{msl} mission with this very purpose.
-Its sophisticated equipment quickly discovered that the conditions to support life as we know it had existed on Mars through chemical and mineral evidence.\cite{marsnasagov_chemcam}
+Thanks to its sophisticated equipment, Curiosity was quickly able to find evidence of past habitable environments on Mars through chemical and mineral evidence.\cite{marsnasagov_chemcam}
```

Comment:
```suggestion
Thanks to its sophisticated equipment, Curiosity was able to find evidence of past habitable environments on Mars through chemical and mineral evidence.\cite{marsnasagov_chemcam}
```

Used quickly in previous sentence

Reply:
Agree

Reply:
We need to mention that it was early in the mission, that's why I used this word. It's crucial to understanding how important this was.